Assets/Scripts/Entity/Player.cs

```diff
@@ -43,7 +44,12 @@ public void OnTriggerEnter2D(Collider2D other)
 
         public void Hurt(Vector2 KnockBack) 
         {
+            int oldHealth = Health;
+
             Health--;
+
+            EventSystem.Current.PlayerHurt(oldHealth, Health);
```

Comment:
Results in the same problem as player death event. Object reference not set.

Reply:
Likely to be fixed with death event implementation.

Reply:
This is cause EventSystem does not exist in that scene